docs/GUIDE.en.md

```diff
@@ -2348,6 +2477,155 @@ near contract \
+ NEAR ABI version                       0.4.0 
+---------------------------------------------------------------------------------------------------------
+
+ Functions: (hint: you can download full JSON Schema using `download-abi` command)
+  
+▹▹▸▹▹ Contract inspection ...  
+▹▹▹▹▸  ↳  Analysis of contract data ...                                                                                                                                                                                             ... Arguments (JSON Schema):
+       {
+         "serialization_type": "json",
+         "args": [
+           {
+             "name": "post_id",
+             "type_schema": {
+               "type": "integer",
+               "format": "uint64",
+               "minimum": 0.0
+             }
+           }
+         ]
+       }
+ ... Return Value (JSON Schema):
+       No return value
+  
+ fn add_member(...) -> ... (read-write function - transcation required)  
+ ... Arguments (JSON Schema):
+       {
+         "serialization_type": "json",
+         "args": [
+           {
+             "name": "member",
+             "type_schema": {
+               "$ref": "#/definitions/Member"
+             }
+           },
+           {
+             "name": "metadata",
+             "type_schema": {
+               "$ref": "#/definitions/VersionedMemberMetadata"
+             }
+           }
+         ]
+       }
+ ... Return Value (JSON Schema):
+       No return value
+  
+ fn add_post(...) -> ... (read-write function - transcation required) payable  
+ ... Arguments (JSON Schema):
+       {
+         "serialization_type": "json",
+         "args": [
+           {
+             "name": "parent_id",
+             "type_schema": {
+               "type": [
+                 "integer",
+                 "null"
+               ],
+               "format": "uint64",
+               "minimum": 0.0
+             }
+           },
+           {
+             "name": "body",
+             "type_schema": {
+               "$ref": "#/definitions/PostBody"
+             }
+           },
+           {
+             "name": "labels",
+             "type_schema": {
+               "type": "array",
+               "items": {
+                 "type": "string"
+               },
+               "uniqueItems": true
+             }
+           }
+         ]
+       }
+ ... Return Value (JSON Schema):
+       No return value
+... ... ...       
+```
+</details>
+
+<details><summary><i>Demonstration of the command in interactive mode</i></summary>
+<a href="https://asciinema.org/a/FKvZYq7vHu0zwvw3KYSKwQuBB?autoplay=1&t=1&speed=2">
+    <img src="https://asciinema.org/a/FKvZYq7vHu0zwvw3KYSKwQuBB.png" width="836"/>
+</a>
+</details>
+
+#### download-abi - Download contract ABI
+
+You can download the contract ABI for the current moment (***now***) and for a certain moment in the past by specifying the block (***at-block-height*** or ***at-block-hash***).
```

Comment:
nit: could add a link to https://github.com/near/abi to try to give more details about what it was, that the command was downloading

Reply:
Done